libraries/lib-effects/CMakeLists.txt

```diff
@@ -19,6 +19,13 @@ set( SOURCES
    PerTrackEffect.h
    StatefulEffectBase.cpp
    StatefulEffectBase.h
+
+   processors/CompressorProcessor.cpp
+   processors/CompressorProcessor.h
+   processors/SimpleCompressor/GainReductionComputer.cpp
+   processors/SimpleCompressor/GainReductionComputer.h
+   processors/SimpleCompressor/LookAheadGainReduction.cpp
+   processors/SimpleCompressor/LookAheadGainReduction.h
```

Comment:
Why do files relating only to one particular effect belong in lib-effects?  I don't agree with the file placement.

Reply:
Because then they can be reused by another target, namely a unit test.

Reply:
It is extraneous to lib-effects.  I would rather give it its own library if you must have I it tests.  Ideally mid-effect-classes could be separated as in my pull request and made independent of the Audacity executable and given its own unit tests, but that presupposes much dependency breaking that is not yet figured out.

Reply:
I would also like to have framework-agnostic processors have their own library, easy to re-use by other projects.
We're only at the beginning of the release cycle, the code around the compressor will not rest until 3.7, I'll have time to relocate.